### PR TITLE
Add applying section with new SoP guide

### PR DESCRIPTION
**Pull request by SolalPirelli**
I also simplified the index page and removed its use of EPFL-specific jargon, so that people who want to apply don't get scared :-)

### CODE REVIEW

File: _applying/why-epfl.md [hunk 1]
```diff
@@ -28,7 +29,7 @@ Note that in Switzerland, virtually nobody stops at the Bachelor level, though s
 How much you want to get involved is up to you, you can do anything from purely back-office stuff like grading to giving lectures or even teaching your own small course.
 This is great if you love teaching or want experience to go into academia.
 
-[Here](https://nebelwelt.net/blog/2021/0121-phd_at_epfl.html) is an excellent guide from one of IC's professors [Mathias Payer](https://people.epfl.ch/mathias.payer).
+[Here](https://nebelwelt.net/blog/2021/0121-phd_at_epfl.html) is an excellent guide from EPFL professor [Mathias Payer](https://people.epfl.ch/mathias.payer).
```
Comment (vinitra) [inline _applying/why-epfl.md:32]: I'd say "EPFL computer science professor"

**Review by vinitra — APPROVED** [review]
Looks great, thanks @SolalPirelli

File: _applying/why-epfl.md (hunk 1 above)
Comment (SolalPirelli) [inline _applying/why-epfl.md:32]:
```suggestion
[Here](https://nebelwelt.net/blog/2021/0121-phd_at_epfl.html) is an excellent guide from EPFL computer science professor [Mathias Payer](https://people.epfl.ch/mathias.payer).
```

Comment (SolalPirelli) [reply _applying/why-epfl.md]: done, thanks

**Pull request merged**